documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -215,6 +252,132 @@ Usage Examples
    1 2 3 3 3 5
    s o r t e d
 
+``radix_sort`` out of place Example
+----------------------
```

Comment:
The header underline needs to go the full length of the header, otherwise it renders incorrectly. Please fix the length.

Reply:
fixed.